### DISCUSSION TITLE
How to create a sealing fault in flow simulation ?

### DISCUSSION THREAD

**Discussion by alexandrebtessael** [GEOS-DEV/GEOS#1860] [Q&A] [answered]
Hello. I am working on an imported mesh which contains several faults to perform multiphaseWell simulations.
The mesh is a .msh file in which I used the PhysicalNames to define a surface region called 'Fault' composed of only quadrangles :

$PhysicalNames
1
1 2 2 "Fault"
$EndPhysicalNames

I want to create sealing faults but I did not find information regarding this topic in your documentation. For example in Eclipse, I used to modify the transmissibility of the faults to obtain this certain behaviour.  I was wondering if you could please tell me how to do this in GEOSX.

**Comment by francoishamon** [comment]
Hello @alexandrebtessael 

If you want to reproduce the Eclipse-style transmissibility multiplier in this simulation, you can add in the `FieldSpecifications` block:

```
  <FieldSpecification                                                                                                                                                                          
      name="barrierTransMult"                                                                                                                                                                    
      objectPath="faceManager"                                                                                                                                                                   
      fieldName="permeabilityTransMultiplier"                                                                                                                                                    
      scale="0"                                                                                                                                                                                  
      initialCondition="1"                                                                                                                                                                       
      setNames="{ Fault }"/> 
```
where `Fault` is the surface tag used in the msh file, and the value of `scale` is the multiplier (0 for a seal).

Let me know if you encounter some issues while doing that.

**Comment by alexandrebtessael** [comment]
Hello @francoishamon and thanks for your answer. I tried your method but it did not change the behaviour of the faults unfortunately.
To try again your method on a smaller model, I created a cube_5x5x5.msh with a vertical fault. I added one well with water injection and one production well but it still produced the same result with water flowing behind the fault (see screenshot below). 

Can you please have a look at my files and tell me if I am missing something to make it work ?
The link to the files is : https://filesender.renater.fr/?s=download&token=fa88e4b6-ca51-4a0f-b0eb-017f58dd87de  

![image](https://user-images.githubusercontent.com/87317386/160825082-a76fa668-6b2c-481e-a633-8f45cc28c984.png)

**Reply by francoishamon** [reply]
I encourage you to use a more recent version of GEOSX, if you can recompile. @klevzoff has enabled the `logLevel` for the mesh import, which you can always leave equal to 1, because it will save you a lot of time debugging this type of problem.

In this case, the log says that PAMELA imports the fault under the name `DEFAULT` (there may be a tagging problem in your file), so you need to write:

```
  <FieldSpecification                                                                                                                                                                          
      name="barrierTransMult"                                                                                                                                                                    
      objectPath="faceManager"                                                                                                                                                                   
      fieldName="permeabilityTransMultiplier"                                                                                                                                                    
      scale="0"                                                                                                                                                                                  
      initialCondition="1"                                                                                                                                                                       
      setNames="{ DEFAULT }"/>
```

and then it will work.

The updated xml is attached:
[cube_multiphaseWell_fault.txt](https://github.com/GEOSX/GEOSX/files/8393331/cube_multiphaseWell_fault.txt)

**Comment by alexandrebtessael** [comment, marked as answer]
Hello @francoishamon , I thought I had responded to your message, thanks a lot ! It did work the way I wanted on my model.
